harvest.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+export RIKOLTI_HOME="/Users/awieliczka/Projects/rikolti"
```

Comment:
Can we add `RIKOLTI_HOME` as an argument to the script? Or just instruct the user to set it in their env before running the script?

Reply:
I'm not trying to spend a whole lot of time on these `harvest.sh` and `harvest_collection.sh` scripts unless folks will use them. It seems like no one used the `rikolti_bucket_temp` branch that I threw up either. The branch has just been lingering, though, gradually getting more out of date, and my local repository has been all kinds of a mess with generating validation reports. This at least folds these two scripts in with the main branch and keeps things moving together, even though the contents of these scripts are hardcoded to be relevant to my machine. Do you think it's worth flushing this out into something more broadly functional here though?